app/models/course.rb

```diff
@@ -124,4 +128,11 @@ def content_status
       :draft
     end
   end
+
+  def ucas_status
+    return :running if findable?
```

Comment:
How come a course is running if it's findable? Are they the same concept?

Reply:
Findable means it shows up on Find / Search and compare.
Running probably means it's on UCAS' Apply system.

Reply:
@benilovj instructed this and can explain better I imagine 😬